misago/richtext/tests/test_markdown.py

```diff
@@ -0,0 +1,159 @@
+from unittest.mock import ANY
```

Comment:
Separate `test_bbcode_blocks` and `test_bbcode_inline` test suites should be implemented like how its done for existing markdown syntax.